code/modules/admin/verbs/panicbunker.dm

```diff
@@ -0,0 +1,16 @@
+/client/proc/panicbunker()
+	set category = "Server"
+	set name = "Toggle Panic Bunker"
+	if (!config.sql_enabled)
+		usr << "<span class='adminnotice'>The Database is not enabled!</span>"
+		return
+
+	if (!dbcon || !dbcon.IsConnected())
```

Comment:
todo: make it so you can turn it off while the database is not connected, or flat out remove some of these checks